src/covid19model/visualization/optimization.py

```diff
@@ -0,0 +1,146 @@
+    filename: string, optional
+        Filename + extension to save a copy of the plot_fit
+    ax : matplotlib.axes.Axes, optional
+        If provided, will use the axis to add the lines.
+
+    Returns
+    -----------
+
+    Notes
+    -----------
+
+    Example use
+    -----------
+
+
+    """
+
+    # Initialize figure and visualize data
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # check if ax object is provided by user
+    if ax is None:
+        fig, ax = plt.subplots()
+    # Create shifted index vector using self.extraTime
+    idx = pd.date_range(startDate,freq='D',periods=data[0].size + model.extraTime) - datetime.timedelta(days=model.extraTime)
+    # Plot data
+    for i in range(len(data)):
+        ax.scatter(idx[model.extraTime:],data[i],color="black")
+
+    # Perform simulation(s) and plot model prediction
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # Compute number of dataseries
+    n = len(data)
+    # Compute simulation time
+    T = data[0].size+model.extraTime-1
+    # Perform simulation(s)
+    if samples is None:
+        k = 0
+        while k < 200:
+            out = model.sim(T,checkpoints=checkpoints)
+            out = out.sum(dim="stratification")
+            for i in range(len(data)):
+                data2plot = out[states[i]].to_array(dim="states").values.ravel()
+                ax.plot(idx,data2plot,linewidth=0.25,alpha=0.2,color="blue")
+            k = k +1
+    else:
+        original_parameters = model.parameters.copy()
+        k=0
+        while k < 100:
+            for key,value in samples:
+                # do random draw and assign to model
+                model.parameters[key] = random.choice(value)
+            # run simulation
+            out = model.sim(T,checkpoints=checkpoints)
+            out = out.sum(dim="stratification")
+            for i in range(len(data)):
+                data2plot = out[states[i]].to_array(dim="states").values.ravel()
+                ax.plot(idx,data2plot)
+            k = k+1
+        # reset parameters
+        model.parameters=original_parameters
+    
+
+    # Attributes
+    if legendText is not None:
+        ax.legend(legendText, loc="upper left", bbox_to_anchor=(1,1))
+    if titleText is not None:
+        ax.set_title(titleText,{'fontsize':18})
+    plt.gca().xaxis.set_major_locator(mdates.DayLocator())
+    plt.gca().xaxis.set_major_formatter(matplotlib.dates.DateFormatter('%d-%m-%Y'))
+    plt.setp(plt.gca().xaxis.get_majorticklabels(),
+        'rotation', 90)
+    ax.set_xlim( idx[model.extraTime-3], pd.to_datetime(idx[-1]+ datetime.timedelta(days=1)))
+    ax.set_ylabel('number of patients')
+    # Hide the right and top spines
+    ax.spines['right'].set_visible(False)
+    ax.spines['top'].set_visible(False)
+    # Only show ticks on the left and bottom spines
+    ax.yaxis.set_ticks_position('left')
+    ax.xaxis.set_ticks_position('bottom')
+    # enable the grid
+    plt.grid(True)
```

Comment:
You add this in earlier plots as well, but this is not required anymore, as these are defined in the updated `rc.param` configuration, see https://github.com/UGentBiomath/COVID19-Model/blob/master/src/covid19model/visualization/output.py#L15-L18